package.json

```
@@ -48,6 +48,7 @@
     "typechain": "^8.1.0"
   },
   "dependencies": {
+    "@chainlink/env-enc": "^1.0.1",
```

```suggestion
    "env-enc": "^1.0.1",
```
(for when published)